sdk/analytics/src/main/java/com/klaviyo/analytics/DeviceProperties.kt

```diff
@@ -11,60 +11,60 @@ import com.klaviyo.core.config.getPackageInfoCompat
 import com.klaviyo.core.model.fetchOrCreate
 import java.util.UUID
 
-internal object DeviceProperties {
+object DeviceProperties {
```

Comment:
total nit: Was this internal object made accessible just so we could `userAgent`? I guess I see why we popped user agent in here since its constructed from so many values tied directly to this class but just had me wondering if this should be moved elsewhere out of the object

Reply:
Good point. Might also be able to do this via registry

Reply:
Ha thats a good point, but honestly a better solution than what I was thinking when I commented. Nice!